### PR TITLE
Adding smuggler.py

### PR DESCRIPTION
**Pull request by fransosiche**
Hello,

As @wlayzz suggested, I added the tool smuggler.py to Exegol (very cool project btw).

### CODE REVIEW

**Comment by ShutdownRepo** [comment]
Nice addition! I'll let you fix the conflict and we'll merge that 👍

**Comment by fransosiche** [comment]
Thx, conflict fixed, should be good !

File: sources/install.sh [hunk 1]
```diff
@@ -3058,6 +3067,7 @@ function install-genusernames() {
   sed -i 's/genadname/genusernames/g' genusernames.function
   echo 'source /opt/tools/genusernames/genusernames.function' >> ~/.zshrc
   add-test-command "genusernames 'john doe'"
+
```
Comment (QU35T-code) [inline sources/install.sh:3070]: Here, there is one empty line.

File: sources/zsh/aliases.d/smuggler [hunk 2]
```diff
@@ -0,0 +1,2 @@
+alias smuggler='python3 /opt/tools/smuggler/smuggler.py'
+alias smuggler.py='python3 /opt/tools/smuggler/smuggler.py'
```
Comment (QU35T-code) [inline sources/zsh/aliases.d/smuggler:2]: There is no newline at the end of the file.

File: sources/zsh/history.d/smuggler [hunk 3]
```diff
@@ -0,0 +1 @@
+smuggler -u "http://$TARGET/"
```
Comment (QU35T-code) [inline sources/zsh/history.d/smuggler:1]: There is no newline at the end of the file.

File: sources/install.sh [hunk 4]
```diff
@@ -3044,6 +3044,15 @@ function install_tshark() {
   add-test-command "tshark --version"
 }
 
+
```
Comment (QU35T-code) [inline sources/install.sh:3047]: There is an extra line between the 2 functions.

**Review by QU35T-code — CHANGES_REQUESTED** [review]
Hello @fransosiche,

Thank you for your participation to the Exegol project !
To be really precise, I propose some small fixes and that will be perfect I think !

**Comment by fransosiche** [comment]
Good point @QU35T-code 

I fixed it !

**Review by ShutdownRepo — APPROVED** [review]

**Comment by ShutdownRepo** [comment]
Thank you for the review @QU35T-code, and @fransosiche for this PR!
Ready to merge, putting it in waitlist because of the current works on the CI/CD pipeline

**Review by QU35T-code — APPROVED** [review]

**Comment by Dramelac** [comment]
Hello @fransosiche

Sorry to repeat the process again but it's the last time!
Can you update your PR with the dev branch to resolve conflicts (this update will also merge the new CI/CD pipeline)

After that we will be able to run the test and merge your PR!

And thank you for your contribution!

**Comment by fransosiche** [comment]
Hello @Dramelac, 

it should be good !

**Pull request merged**